src/instruments/src/ND/pages/ArcMode.tsx

```diff
@@ -633,7 +636,7 @@ const SelectedHeadingBug: React.FC<{heading: number, selected: number}> = ({ hea
             className="Cyan"
             fontSize={22}
         >
-            {`${("" + Math.round(selected)).padStart(3, "0")}`}
+            {`${selected}`}
```

Comment:
This will cause a whole bunch of decimal places to show when the autopilot reverts from managed to selected heading, and also no longer zero-padding to 3 digits...